courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/3_keras_sequential_api.ipynb

```diff
@@ -156,6 +156,13 @@
     "We use a python dictionary comprehension to create the feature columns for our model, which is just an elegant alternative to a for loop."
```

Comment:
<p>Pass this function to below to the model -&gt; Pass this function to the model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Great exercises! Just minor comments. </p>

Reply:
<p>done.</p>